### DISCUSSION TITLE
Failure to login to your email account!

### DISCUSSION THREAD

**Discussion by MattLogue** [berrystephenw/OctoText#60] [General]
I can't not get this thing to work. I tried Outlook just as shown. Then I disabled google 2step and enabled less-secure apps. Still no luck. Now I am using an app password that I copied and pasted for google and it still fails. Well shit looked in the log and it's still trying to connect to Outlook. I changed the damned smtp server.
...
smtplib.SMTPAuthenticationError: (535, b'5.7.3 Authentication unsuccessful [CH2PR10CA0027.namprd10.prod.outlook.com]')
Going to try to reboot the RPi and octoprint.... oh good now it timed out because I put in smtp.fuckyou.com in... so we're getting somewhere. Guess you have to reboot once you change smtp server. Odd bug.

Holy shit it works!

**Comment by berrystephenw** [comment]
That was a wild ride reading your comments!
One thing that caught me developing the plug-in is remembering to hit the save button in the settings form. If you don't do that it reverts to the original settings. You shouldn't need to reboot.
If you have any more trouble let me know.

**Comment by MattLogue** [comment]
Thx man it did throw me for a loop but figured it out. Now only if it could
notify based on layer.... 😁 I could use m73 or whatever to set % and force
it to notify. Tired of watching for hours to change filament at layers...

**Reply by berrystephenw** [reply]
I think PrusaSlicer will do what you want. You can set the color change at a particular layer and it will pause the printer - oh, that might only be for their printers. Might be worth looking into if you haven't already.
